libvast/vast/system/source.hpp

```diff
@@ -126,6 +126,12 @@ struct source_state {
   measurement measurement_;
 
   void send_report() {
+    // Send the reader-specific status report to the accountant.
+    if (auto status = reader.status(); !status.empty()) {
+      if (accountant)
+        self->send(accountant, std::move(status));
+    }
+    // Send the source-specific performance metrics to the accountant.
```

Comment:
Would it make sense to consolidate the reporting into a single message?

Reply:
One is a `report` and the other one is a `performance_report`.